README.md

```diff
@@ -575,6 +578,30 @@ $client->packages()->listCustomers('acme-website/package');
 ```
 Returns a list of customers with access to the package.
 
+#### Create an artifact package file
+
+```php
+$fileName = 'package1.zip'; // your package archive artifact containing a valid composer.json in root directory
+$file = file_get_contents($fileName);
+$client->packageArtifact()->create($file, 'application/zip', $fileName);
+```
+
+#### Create an artifact package
+
+```php
+$fileName = 'package1.zip';
+$file = file_get_contents($fileName);
+$response = $client->packageArtifact()->create($file, 'application/zip', $fileName);
+$artifactId = $response['id'];
+$client->packages()->createArtifactPackage([$artifactId]);
+```
+#### Update an artifact package files
+
+```php
+$result = $client->packages()->showArtifacts('acme-website/package'); // get artifact files details for a package
```

Comment:
Shouldn't this method go to the `Artifacts` client class?

Reply:
something like `$client->packages()->artifacts()->show('acme-website/package')` (and same for edit) ? I agree I think this seems better

Reply:
yes